internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/DataRecoveryTest.java

```diff
@@ -311,7 +311,7 @@ public void testHybridAADataRecovery() throws Exception {
                   storeName,
                   new UpdateStoreQueryParams().setHybridRewindSeconds(10)
                       .setHybridOffsetLagThreshold(2)
-                      .setHybridDataReplicationPolicy(DataReplicationPolicy.ACTIVE_ACTIVE)
+                      .setHybridDataReplicationPolicy(DataReplicationPolicy.NON_AGGREGATE)
```

Comment:
Why not use NONE?

Reply:
The following check won't pass with NONE.
```
    if (pushType.equals(PushType.STREAM)
        && store.getHybridStoreConfig().getDataReplicationPolicy().equals(DataReplicationPolicy.NONE)) {
      throw new VeniceHttpException(
          HttpStatus.SC_BAD_REQUEST,
          "requesting topic for streaming writes to store " + store.getName()
              + " which is configured to have a hybrid data replication policy "
              + store.getHybridStoreConfig().getDataReplicationPolicy(),
          ErrorType.BAD_REQUEST);
    }
```


I wonder if we should just replace it with `    if (pushType.equals(PushType.STREAM)
        && (store.getHybridStoreConfig().getDataReplicationPolicy().equals(DataReplicationPolicy.NONE) && !store.isAaEnabled())) {`

Reply:
```
  /**
   * This enum value is used in 2 cases:
   *   1. batch-only stores since it has no real-time data replication.
   *   2. incremental push stores that do not have Samza jobs.
   */
  NONE(2),
```

It doesn't make sense for batch-only stores as they won't have a hybrid config (where DRP lives). So, it's only the 2nd one.

Let's not increase the scope of DRP NONE. I think we should be able to remove that too. But it is most-likely out of scope of this change

Reply:
>Let's not increase the scope of DRP NONE. I think we should be able to remove that too. But it is most-likely out of scope of this change

Do you recommend using NON_AGG over NONE whenever possible?

Reply:
Reading through the PR again, it looks okay to change DRP to NONE, and have the handling you suggested originally. I seem to prefer that over `NON_AGGREGATE` for AA stores 😅. Can you please update the javadoc of DRP NONE also?

Reply:
sure